examples/backends/05_slurm_backend_run.py

```diff
@@ -0,0 +1,352 @@
+            qos="high",                     # Quality of Service
+            reservation="special_reservation",  # Reservation name
+            output_file="slurm-%j.out",     # Output file pattern (job ID)
+            error_file="slurm-%j.err",      # Error file pattern
+            job_name="advanced_simulation", # Name of the SLURM job
+            mail_type="BEGIN,END,FAIL",     # Types of notifications
+            mail_user="researcher@domain.com",  # Email for notifications
+            additional_options=["--gres=gpu:v100:2", "--exclusive"],  # GPU resources
+            env_vars={                      # Environment variables
+                "OMP_NUM_THREADS": "4",
+                "MODEL_DEBUG": "true",
+                "DATA_PATH": "/shared/data",
+                "RESULTS_PATH": "/shared/results",
+            },
+        )
+
+        logger.info(f"Advanced SlurmConfig created: {config}")
+        logger.info("Running model with advanced SLURM configuration...")
+
+        try:
+            # Show validation success
+            logger.info("✅ Advanced SlurmConfig validated successfully")
+            logger.info("Key concepts: account, qos, reservations, GRES, environment variables")
+            logger.info("Note: In a real environment, this would submit a complex job to SLURM")
+        except Exception as e:
+            logger.error(f"❌ Advanced SLURM configuration failed: {e}")
+
+
+def example_slurm_with_custom_command():
+    """
+    Example 3: SLURM execution with custom command
+    
+    This example shows how to run a custom command on the SLURM cluster,
+    useful for executing different types of jobs or calling external binaries.
+    """
+    logger.info("=" * 60)
+    logger.info("Example 3: SLURM with Custom Command")
+    logger.info("=" * 60)
+    logger.info("This example demonstrates running custom commands on SLURM.")
+    logger.info("")
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        model = ModelRun(
+            run_id="slurm_custom_command_example",
+            period=TimeRange(
+                start=datetime(2023, 1, 1),
+                end=datetime(2023, 1, 2),
+                interval="1H",
+            ),
+            output_dir=Path(temp_dir),
+            delete_existing=True,
+        )
+
+        # SLURM configuration with a custom command
+        config = SlurmConfig(
+            queue="general",
+            timeout=3600,  # 1 hour timeout
+            nodes=1,
+            ntasks=1,
+            cpus_per_task=2,
+            time_limit="01:00:00",
+            command="echo 'Running custom SLURM job' && date && pwd && ls -la",  # Custom command
+            env_vars={"CUSTOM_VAR": "value"},
+        )
+
+        logger.info(f"SlurmConfig with custom command: {config}")
+        logger.info("Running custom command on SLURM...")
+
+        try:
+            logger.info("✅ SlurmConfig with custom command validated successfully")
```

Comment:
This example doesn't seem to do much, just check if the constructor doesn't error out.